.gitpod.yml

```
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: pnpm install && pnpm run build
+    command: pnpm run start
```

```suggestion
tasks:
  - init: pnpm install
    command: pnpm run start
```
Just checking whether you intended this, or even to commit this at all, because I know this is usually the typical Gitpod start up file when you don't have one.

It doesn't matter as much *now* when your site is small, but I have my `gitpod.yml` specifically set NOT to build the docs site when I open it because of how long it takes!!  😅

---

I just added everything from the GitPod, without looking at this file, not sure if we want it or not. Might be helpful if we use GitPod in the future too.. 

cc @florian-lefebvre ??

---

i'm not familiar with gitpod at all, if this file helps us start more gitpod sessions in the future then let's keep it

---

@sarah11918 can you enlighten us about GitPod a little bit?

---

Yes I can!

When you start a Gitpod, this file will be created for you with their defaults. If you just blanket commit everything, then you commit this file.

So you don't *need* this file at all! Gitpod will always create a new one for you! But, in doing so, it's always Gitpod's default which is to both install and build your project. (Which, for docs, is a BAD IDEA because of how long it takes.)

That's why docs *does* use a file, not to make it work, but rather to override the defaults when you don't have one.

---

Ok then I think it makes sense to keep it :)